code/AmazonScraper.py

```diff
@@ -68,7 +68,7 @@ def check_stock(self, url):
             soup = BeautifulSoup(page.text, "html.parser")
         # Handles invalid URLs/timeouts
         except:
-            return "Error Occurred"
+            return "Error Occurred", "NA"
```

Comment:
Since this function is named and documented such that it only the stock information is returned, we can either create a new function that returns the cost, or make this a wrapper function and get the cost and stock separately. 
As a quick fix we can just rename the function and modify the function documentation.

Reply:
How about we go for the quick fix here ?